src/Controller/CRUDController.php

```diff
@@ -1307,7 +1335,7 @@ protected function handleXmlHttpRequestSuccessResponse(Request $request, object
             'result' => 'ok',
             'objectId' => $this->admin->getNormalizedIdentifier($object),
             'objectName' => $this->escapeHtml($this->admin->toString($object)),
-        ], Response::HTTP_OK);
+        ]);
     }
 
     final protected function assertObjectExists(Request $request): void
```

Comment:
What should I do inside the `checkParentChildAssociation` method ?
- if `$this->admin->getParentAssociationMapping()` is null ?
- if `$propertyAccessor->getValue($object, $propertyPath);` is null ?